pkg/flowkit/gateway/emulator.go

```diff
@@ -57,21 +56,21 @@ func NewEmulatorGatewayWithOpts(serviceAccount *flowkit.Account, opts ...func(*E
 
 	gateway := &EmulatorGateway{
 		ctx:             context.Background(),
-		logger:          logrus.New(),
+		logger:          &zerolog.Logger{},
 		emulatorOptions: []emulator.Option{},
 	}
 	for _, opt := range opts {
 		opt(gateway)
 	}
 
 	gateway.emulator = newEmulator(serviceAccount, gateway.emulatorOptions...)
-	gateway.backend = backend.New(&zerolog.Logger{}, gateway.emulator)
+	gateway.backend = backend.New(gateway.logger, gateway.emulator)
```

Comment:
Unrelated: 
  Hm... Interesting that zerolog is passed by reference. I'm not sure it should be. Something to look into...

Reply:
hmm I was thinking on this when moving from logrus to zerolog, but I felt reference passing is correct. But I maybe totally missing the point. Why do you think copy is better @janezpodhostnik  ?

Reply:
I think the issue might be that the receiver can have an influence on the logging of the other components.

Reply:
In any case I don't think this is the subject of this PR

Reply:
I think in that case pass by reference is still correct. Sender always have option to send a new logger, but on the other case, sender loses influence on the component. ( e.g. change logging level in runtime )